scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatWriter.scala

```diff
@@ -559,7 +559,11 @@ class FormatWriter(formatOps: FormatOps) {
   def alignmentTokens(
       locations: Array[FormatLocation]
   ): Map[TokenHash, Int] = {
-    if (initStyle.align.tokens.isEmpty || locations.length != tokens.length)
+    lazy val noAlignTokens = locations.forall { floc =>
```

Comment:
1. i'd just `def` instead of `lazy val`, if the value is used only once.
2. rather than iterating over locations and looking up in `styleMap`, why not iterate over `styleMap.values` directly?

Reply:
It may be simply `val`. `lazy` is here because `noAlignTokens` is second operand of `||` and will not computed if `locations.length != tokens.length` is true. But I doubt that this is a significant optimization, so it can be just `val`.

> why not iterate over styleMap.values directly?

How iterate it here? Size of `styleMap` look bigger than `locations` size

Reply:
if `styleMap` is larger, then yes. but i thought that `styleMap` was only for token ranges for which any custom overrides were specified, no? so in most cases (since override is rare), the map itself would be empty, leaving just `initStyle`.

Reply:
Yes, you are right